packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -127,7 +127,7 @@ export interface IContainerRuntimeBase extends
      */
     getAbsoluteUrl(relativeUrl: string): Promise<string | undefined>;
 
-    getTaskManager(): Promise<ITaskManager>;
+    getScheduler(): Promise<IAgentScheduler>;
```

Comment:
>getScheduler [](start = 4, length = 12)

do we even want to expose this? a user can always just request it

Reply:
Good question, and that is what FFX is doing currently.  It does offer a (suggested) choke point where we can alert the customer to the eventual deprecation in the future which is kind of nice, but maybe it would be better to avoid giving easy access at all.  I'll think about this a bit.

Reply:
especially considering it's being removed soon?

Reply:
Removed in latest